bundle/Cache/SwitchableHttpCachePurger.php

```diff
@@ -42,12 +42,12 @@ public function purgeAll()
         $this->gatewayCachePurger->purgeAll();
     }
 
-    public function purgeForContent($contentId)
+    public function purgeForContent($contentId, $locationIds = array())
```

Comment:
Isn't this a breaking change? If so, we should require eZ Kernel `>=6.3.2|>=6.4.2` (when 6.4.2 is released) in `composer.json`, right?